src/common/helpers/data-extract/exotics-application.test.js

```diff
@@ -0,0 +1,145 @@
+import { ExoticsApplication } from './exotics-application.js'
+
+describe('ExoticsApplication', () => {
+  let mockApplicationData
```

Comment:
`const mockApplicationData = { ... }`

would be preferable.  This data should never be mutated, so our tests reusing the same exact object will help reveal if we have.